controllers/comments-controller.js

```diff
@@ -17,14 +19,21 @@ exports.getComments = (req, res, next) => {
 };
 
 exports.postComment = (req, res, next) => {
-    const commentToPost = req.body;
-    const { article_id } = req.params;
-   createComment(commentToPost, article_id)
-     .then((comment) => {
-       res.status(201).send({ comment: comment });
-     })
-     .catch((err) => {
-       next(err);
-     });
+  const commentToPost = req.body;
+  const { username } = commentToPost;
+  const { article_id } = req.params;
+  const promises = [
+    createComment(commentToPost, article_id),
+    //checkUserExists(username),
```

Comment:
Don't forget to remove any unused code

Reply:
You don't have to use promises here. The fact one of the promises is a simple value suggests we could maybe just check our rows are 0 and reject accordingly